.circleci/config.yml

```diff
@@ -92,5 +130,5 @@ workflows:
   netlify_manual:
     when: << pipeline.parameters.trigger_netlify_manual >>
```

Comment:
Will writers need to enter trigger_netlify_manual when they trigger a preview build?
If so could we simplify it to simply `preview`?

Reply:
Good suggestion. Changed it to `preview`.